vlasovsolver/cpu_acc_transform.cpp

```diff
@@ -123,6 +123,26 @@ Eigen::Transform<Real,3,Eigen::Affine> compute_acceleration_transformation(
       spatial_cell->parameters[CellParams::EYGRADPE],
       spatial_cell->parameters[CellParams::EZGRADPE]);
 
+   // If a bulk velocity is being forced here, perform that first, and let everything below continue in the new frame.
+   if(spatial_cell->parameters[CellParams::FORCING_CELL_NUM] > 0) {
+      Eigen::Matrix<Real,3,1> forced_bulkv(spatial_cell->parameters[CellParams::BULKV_FORCING_X],
+                                           spatial_cell->parameters[CellParams::BULKV_FORCING_Y],
+                                           spatial_cell->parameters[CellParams::BULKV_FORCING_Z]);
+      forced_bulkv /= spatial_cell->parameters[CellParams::FORCING_CELL_NUM];
+
+      Eigen::Matrix<Real,3,1> bulkDeltaV = forced_bulkv - bulk_velocity;
+      total_transform=Translation<Real,3>(bulkDeltaV) * total_transform;
```

Comment:
I think the actual rotation has to take place in the Hall frame still, so the forcing should be added after everything else to just nudge the VDF into the correct bulk velocity. Otherwise looks preliminarily sane, as sane as this forcing can be!

Reply:
Ah, you mean that otherwise we'll get the whole distribution agyrotropic and rotate it by a bit of a gyration fraction, until we drag it back to the forced bulkv frame in the next timestep? Hmm, I guess I imagined that upside down. In that case: yep, let's do the forcing after rotation.